Fluid/Ast/IncludeStatement.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.IO;
+using System.Text.Encodings.Web;
+using System.Threading.Tasks;
+using Microsoft.Extensions.FileProviders;
+
+namespace Fluid.Ast
+{
+    public class IncludeStatement : Statement
+    {
+        private static readonly IFileProvider _blankFileProvider = new NullFileProvider();
```

Comment:
Remove this line, use `TemplateContext.GlobalFileProvider` that you have created.